proto/tendermint/consensus/types.proto

```diff
@@ -85,7 +85,7 @@ message Message {
     ProposalPOL   proposal_pol    = 4;
     BlockPart     block_part      = 5;
     Vote          vote            = 6;
-    HasVote       has_vote        = 7;
+    HasVote       have_vote       = 7;
```

Comment:
We will need to regenerate the protobuf files to accommodate this change. Is there any way cpp could disable generating the code that causes the issue?

Reply:
I afraid can't do that, because that is Protobuf rules. How about I rename some variable in message.go?